src/apps/containers/components/ContainerVersionForm.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useRef } from "react";
+import {
+  Button,
+  ButtonGroup,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Typography
+} from "@material-ui/core";
+import { usePrevious } from "../../../utils";
+import { Field, Form, Formik } from "formik";
+import * as Yup from "yup";
+import { Select, TextField } from "formik-material-ui";
+
+import { Version } from "../../../utils";
+import uuid from "uuid";
+
+interface Props {
+  onSubmit: Function;
+  loading: boolean;
+  error?: { detail: string };
+  handleClose: () => void;
+}
+
+const DictionaryVersionSchema = Yup.object().shape<Version>({
```

Comment:
Hope it should be renamed as `ContainerVersionSchema`

Reply:
Done